app/pages/sections/order-items/catalogue-solutions/order-item/contextCreator.js

```diff
@@ -15,13 +15,17 @@ export const getContext = ({
   selectedPrice,
   selectedRecipients,
   errorMap,
+  catalogueItemAlreadyExist,
 }) => {
   const errorMessages = errorMap && (errorMap.quantity || errorMap.deliveryDate)
     ? ((errorMap.quantity || {}).errorMessages || [''])
       .concat((errorMap.deliveryDate || {}).errorMessages) : undefined;
   const newbackLinkRef = selectedPrice.provisioningType === 'Patient'
     ? `${baseUrl}/organisation/${orderId}/catalogue-solutions/select/solution/price/recipients/date`
     : `${baseUrl}/organisation/${orderId}/catalogue-solutions/select/solution/price/${selectedPrice.type.toLowerCase()}/${selectedPrice.provisioningType.toLowerCase()}`;
+  const backLinkIfSolutionExist = catalogueItemAlreadyExist !== undefined
+    ? `${baseUrl}/organisation/${orderId}/catalogue-solutions/select/solution/`
+    : `${baseUrl}/organisation/${orderId}/catalogue-solutions`;
```

Comment:
I think `newItemBackLink` and `existingItemBackLink` might be slightly better names.

Reply:
changed to existingItemBackLink